File: 3st/Jaeyoung/가위 바위 보.java
class Solution {
    public String solution(String rsp) {
        String answer = "";
        String[] arr = rsp.split("");
        
        for (int i = 0; i < arr.length; i++) {
            if (arr[i].equals("2")) {
                answer += "0";
            } else if (arr[i].equals("0")) {
                answer += "5";
            } else {
                answer += "2";
            }
        }
        return answer;
    }
}

File: 3st/Juram/programmer/가위 바위 보.js
function solution(rsp) {
    let answer = [];
    rsp.split();
    for(let i=0; i<rsp.length; i++){
        if(rsp[i]=="2") answer.push("0");
        else if(rsp[i]=="5") answer.push("2");
        else answer.push("5");
    }
    return answer.join("");
}